src/walletHandlers/verifyHandler.js

```
@@ -43,52 +45,63 @@ module.exports = async (from_address, data) => {
         const { signed_message, authors: [{ address: walletAddress }] } = objSignedMessage;
 
         try {
-            const signedData = JSON.parse(signed_message);
+            logger.error('signed_message', signed_message);
+            const signedData = JSON.parse(signed_message.trim());
+            logger.error('signedData', signedData)
+            const { message, ...data } = signedData;
+            logger.error('data', message, data);
+            let address = data.address;
+
+            if (message && message.includes('I own the address:')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)